subprojects/snapshots/snapshots.gradle.kts

```diff
@@ -45,6 +45,11 @@ dependencies {
     testRuntimeOnly(project(":workers"))
     testRuntimeOnly(project(":dependencyManagement"))
 
+    integTestRuntimeOnly(project(":apiMetadata"))
+    integTestRuntimeOnly(project(":kotlinDsl"))
+    integTestRuntimeOnly(project(":kotlinDslProviderPlugins"))
+    integTestRuntimeOnly(project(":kotlinDslToolingBuilders"))
+
```

Comment:
:o: I would move those `integTest*` dependencies below all `test*` declarations

Reply:
They are underneath all `test*` dependencies, the ones below are for test fixtures. I'd leave it like this unless you feel strongly.

Reply:
Well, we don't have `integTestFixtures*` yet but we should, so I would move the `integTest*` dependencies below

Reply:
OK, I interpret this as you feel strongly. I moved them to the bottom.